css/style.css

```diff
@@ -543,5 +1020,228 @@
   .arrow-downd:hover {
     transform: scale(1.2);
   }
-}
 
+  .future-drive {
+    display: flex;
+    min-height: 900px;
+    background-image: url(/img/2013-tesla-model-s-cockpit.png);
+    background-repeat: no-repeat;
+    background-position-y: 56px;
+  }
+
+  .future-drive__title {
+    margin: 0;
+    padding-top: 70px;
+    padding-left: 80px;
+    font-size: 70px;
+    line-height: 85px;
+    color: #e51937;
+  }
+
+  .future {
+    margin-left: auto;
+    width: 440px;
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .container-future {
+    margin: 0 auto;
+    max-width: 316px;
+  }
+
+  .tittle-article {
+    margin-bottom: 5px;
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .article-text {
+    max-width: 300px;
+    font-size: 15px;
+    line-height: 18px;
+    font-weight: lighter;
+    margin-bottom: 100px;
+  }
+
+  .future-drive__img {
+    display: none;
+  }
+
+  .future-drive-list {
+    margin-bottom: 150px;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  .future-drive-list__item {
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+  }
+
+  .fruture-text {
+    margin: 0;
+    font-size: 35px;
+    line-height: 43px;
+    font-weight: bold;
+  }
+
+  .fruture-text-description {
+    margin: 0;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: normal;
+  }
+
+  .fruture-description {
+    max-width: 140px;
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: lighter;
+  }
+
+  .fruture-text-description-wifi {
+    margin: 0;
+    max-width: 150px;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: normal;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6120308)